code/lib/core-common/src/presets.ts

```diff
@@ -72,6 +72,7 @@ export const resolveAddonName = (
   options: any
 ): ResolvedAddonPreset | ResolvedAddonVirtual | undefined => {
   const r = name.startsWith('/') ? safeResolve : safeResolveFrom.bind(null, configDir);
+  const ifResolves = (path: string) => r(path) && path;
```

Comment:
What about renaming to getResolvedPath?

Reply:
If I'm understanding correctly, this doesn't return the resolved path, it checks if the resolved path is truthy, and then returns the original path, or `undefined`.

Reply:
Ah I see. I read it quickly and didn't notice that 🤦 thanks!